Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
 authors = ["Matthias Endler <matthias-endler.de>"]
-description = "A boring link checker for my projects (and maybe yours)"
-documentation = "https://github.com/hello-rust/lychee/blob/master/README.md"
+description = "A glorious link checker"
```

Comment:
:rocket:

Reply:
Haha, the world shall know about the glory that is this link checker. 🤣